Gemfile.lock

```diff
@@ -144,14 +144,16 @@ GEM
     hashdiff (1.0.1)
     http-cookie (1.0.5)
       domain_name (~> 0.5)
-    i18n (1.10.0)
+    i18n (1.12.0)
       concurrent-ruby (~> 1.0)
     jquery-rails (4.4.0)
       rails-dom-testing (>= 1, < 3)
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
+    json (2.6.2)
     kgio (2.11.4)
     libv8 (3.16.14.19)
+    libv8 (3.16.14.19-x86_64-darwin-18)
```

Comment:
does this mean that the code only works on a mac? because the deploy is failing now